### ISSUE TITLE
ArrayIndexOutOfBoundsException in Object Indicators plugin

### ISSUE THREAD

**Issue by Nightfirecat** [runelite/runelite#15024]
### Bug description

Found in my log from yesterday, though I'm not sure what activity caused it.

```
java.lang.ArrayIndexOutOfBoundsException: Index -1 out of bounds for length 4
	at net.runelite.api.coords.WorldPoint.fromLocalInstance(WorldPoint.java:201)
	at net.runelite.client.plugins.objectindicators.ObjectIndicatorsPlugin.checkObjectPoints(ObjectIndicatorsPlugin.java:272)
	at net.runelite.client.plugins.objectindicators.ObjectIndicatorsPlugin.onGameObjectSpawned(ObjectIndicatorsPlugin.java:156)
	at net.runelite.client.eventbus.EventBus$Subscriber.invoke(EventBus.java:70)
	at net.runelite.client.eventbus.EventBus.post(EventBus.java:223)
	at net.runelite.client.callback.Hooks.post(Hooks.java:169)
	at gn.fu(gn.java:38992)
	at gm.b(gm.java:47207)
	at gm.ru(gm.java:264)
	at h.y(h.java:421)
	at bn.v(bn.java:337)
	at kr.hx(kr.java:5893)
	at client.ag(client.java:1053)
	at aj.h(aj.java:368)
	at aj.run(aj.java:347)
	at java.base/java.lang.Thread.run(Thread.java:829)
```

**Comment by cjhetzle** [comment]
Sounds like fun. I'll give it a shot.

**Comment by Adam-** [comment]
fixed in b8314133d1510bdb66c3440b17b451433a21499d
